src/dnf-plugins/subscription-manager.py

```diff
@@ -142,7 +142,17 @@ def _warnOrGiveUsageMessage(self):
             if msg:
                 logger.info(msg)
 
+    def sack(self):
```

Comment:
I had to use the sack hook as changes in Command.config are not visible in config hook(Propably bug). I talked about this with DNF team and they suggested to use Python's destructor since we need to run the profile action at the end of the command. What do you think?

Reply:
We don't necessarily need to run the profile action command at the end of the command, we only need to do that if there will be no transaction.

Are you sure that running the package_profile upload that late will not end up using stale data?

Reply:
I haven't found a way to detect whether the command will run transaction in config hook. The sack phase is the soonest I can detect it.